builtin/providers/aws/resource_aws_rds_cluster_test.go

```diff
@@ -238,18 +314,37 @@ resource "aws_rds_cluster" "default" {
   master_username = "foo"
   master_password = "mustbeeightcharaters"
   db_cluster_parameter_group_name = "default.aurora5.6"
+  skip_final_snapshot = true
   tags {
     Environment = "production"
   }
 }`, n)
 }
 
+func testAccAWSClusterConfigWithFinalSnapshot(n int) string {
+	return fmt.Sprintf(`
+resource "aws_rds_cluster" "default" {
+  cluster_identifier = "tf-aurora-cluster-%d"
+  availability_zones = ["us-west-2a","us-west-2b","us-west-2c"]
+  database_name = "mydb"
+  master_username = "foo"
+  master_password = "mustbeeightcharaters"
+  db_cluster_parameter_group_name = "default.aurora5.6"
+  skip_final_snapshot = true
```

Comment:
In hindsight, a `testAccAWSClusterConfigWithFinalSnapshot` config should probably have `skip_final_snapshot = false` 😜